src/connectors/deutschlandfunknova.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
-Connector.playerSelector = '.player__main';
+Connector.playerSelector = 'div.player.desktop';
 
-Connector.artistTrackSelector = 'span.title--title';
+Connector.artistTrackSelector = '.playing-title';
 
 Connector.getArtistTrack = () => {
 	let text = $(Connector.artistTrackSelector).text();
-	let m = text.match(/"(.*)" von (.*)/);
+	let m = text.match(/"(.*?)" von (.*?)\u2003\u00B7\u2003/);
```

Comment:
You really need those chars exactly? Wouldn't `.*` be better?

Reply:
The title info is available as ticker text only, like that:
"Title" von Artist · "Title" von Artist · "Title" von Artist
where the separator (' · ') is formed by the special unicode chars (em space, middle dot, em space).
As everything is contained more than once in one line, I think the easiest solution is to use the ungreedy modifier.

Reply:
Ahhh - Okays that makes sense :+1: